tools/lsp/ui/api.slint

```diff
@@ -169,6 +169,8 @@ export global Api {
     in property <[string]> known-styles;
     // The current style
     in-out property <string> current-style;
+    in-out property <string> control-key-name: "control";
+    // control, but command on macOS
```

Comment:
The comment is in the wrong line?

Reply:
Argh, thanks for catching. Format fooled me :). I'll push a quick fix.